src/components/Tx/SendTx/CurrencySelect.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Identicon from '../../Identicon'
+
+export default class CurrencySelect extends Component {
+  static displayName = 'CurrencySelect'
+
+  static propTypes = {
+    etherWallet: PropTypes.object,
+    onClick: PropTypes.func,
+    tokens: PropTypes.array
+  }
+
+  static defaultProps = {
+    etherWallet: {
+      name: 'ETHER',
+      symbol: 'ETHER',
+      address: '0x123456BfA2A4DFAbdD1Fad5A1cb73a63345zzzzz',
+      balance: '0.00'
+    },
+    tokens: []
+  }
+
+  chooseCurrency = e => {
+    const { onClick } = this.props
+    if (onClick) {
+      onClick(e)
+    }
+  }
+
+  static renderCurrencySpans = currency => {
+    return (
+      <React.Fragment>
+        {currency.name === 'ETHER' ? (
```

Comment:
I like minimizing the use of ternary operators. In this case, define the variable above the return and do if or case statements to define it correctly, then just output the variable in the template